kubernetes/k8s-listener.py

```diff
@@ -288,24 +298,45 @@ def parse_rule_specs(obj):
         return None
 
 def get_tenants():
+    """
+    Returns romana tenants description
+
+    Example:
+        [{"Id":1,"Name":"t1","Segments":null,"Seq":1},{"Id":2,"Name":"t2","Segments":null,"Seq":2}]
+    """
     r = requests.get(tenant_url)
     print 'Tenants service returned %s' % r.content
     return simplejson.loads(r.content)
 
 def get_tenant_id_by_name(name, tenants):
+    """
+    Returns romana tenant id
+    Example : 1
+    """
     for tenant in tenants:
         if tenant['Name'] == name:
             return tenant['Id']
 
 def get_segments(tenant_id):
+    """
+    Returns a list of romana segments for particular tenant
+
+    Example:
+    [{"Id":1,"TenantId":1,"Name":"default","Seq":1},{"Id":3,"TenantId":1,"Name":"frontend","Seq":2},{"Id":4,"TenantId":1,"Name":"backend","Seq":3}]
+    """
     r = requests.get(tenant_url + '/' + str(tenant_id) + '/segments')
     return simplejson.loads(r.content)
 
 def get_segment_id_by_name(name, segments):
+    """
+    Returns romana segment id
+    Example : 1
+    """
     for segment in segments:
         if segment['Name'] == name:
             return segment['Seq']
 
+# TODO should probably discover this from romana root
 addr_scheme = {
```

Comment:
When we merge this we just need to make sure that this hard-coded address scheme is matched by the other parts of the installation.